carbon-design-system/carbon-addons-iot-react
# fix(emptystate): adjust proptypes schema

Closes #3370 

**Summary**

- Change PropTypes schema for `EmptyState`

**Change List (commits, features, bugs, etc)**

- PropTypes schema
- Public API snapshot update

**Acceptance Test (how to verify the PR)**

- Go to [this commit](https://github.com/carbon-design-system/carbon-addons-iot-react/pull/3567/commits/0aeda1e8d8aba1b231e2904dc82f60a6d6ce261c)
- Verify that PropTypes fields were converted from single `node` to `oneOfType([PropTypes.node, PropTypes.string])`

**Regression Test (how to make sure this PR doesn't break old functionality)**

- tests here



**Things to look for during review**
